compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmLower.kt

```diff
@@ -289,6 +290,8 @@ private val jvmFilePhases = listOf(
 
     moveOrCopyCompanionObjectFieldsPhase,
     inlineCallableReferenceToLambdaPhase,
+    functionReferencePhase,
```

Comment:
Maybe add `functionReferencePhase` as a `prerequisite` for `propertyReferencePhase`?

Reply:
Done, `suspendLambdaPhase` should be there too I guess.